libraries/radwordsprocessing/formats-and-conversion/rtf/rtfformatprovider.md

```diff
@@ -83,6 +92,9 @@ You can also export the document to a string and preserve it in a database.
 #### __[C#] Example 4: Export a document to a string__
 
 {{region cs-radwordsprocessing-formats-and-conversion-rtf-rtfformatprovider_3}}
+	using Telerik.Windows.Documents.Flow.FormatProviders.Rtf;
+	using Telerik.Windows.Documents.Flow.Model;
```

Comment:
Do not add usings to the snippet - it won't build when directly pasted. Instead, use full class names.